src/main/kotlin/com/ampnet/projectservice/service/impl/OrganizationServiceImpl.kt

```diff
@@ -64,8 +67,14 @@ class OrganizationServiceImpl(
     }
 
     @Transactional(readOnly = true)
-    override fun findAllOrganizationsForUser(userUuid: UUID): List<Organization> {
-        return organizationRepository.findAllOrganizationsForUserUuid(userUuid)
+    override fun findAllOrganizationsForUser(userUuid: UUID): List<OrganizationWitProjectCountServiceResponse> {
+        val organizations = organizationRepository.findAllOrganizationsForUserUuid(userUuid)
+        val projectsMap = projectRepository.findAllByOrganizations(organizations.map { it.uuid })
+            .groupBy { it.organization }
+        return organizations.map { organization ->
+            val projectCount = projectsMap[organization]?.size ?: 0
+            OrganizationWitProjectCountServiceResponse(organization, projectCount)
+        }
```

Comment:
```suggestion
        val projectsMap = projectRepository.findAllByOrganizations(organizations.map { it.uuid })
            .groupBy { it.organization.uuid }
        return organizations.map { organization ->
            val projectCount = projectsMap[organization.uuid]?.size ?: 0
            OrganizationWitProjectCountServiceResponse(organization, projectCount)
        }
```

It is better to have the only UUID for the key.